demos/vc_issuer/src/consent_message.rs

```diff
@@ -0,0 +1,107 @@
+//! This module contains the various consent messages that is displayed to the user when they are asked to consent to the issuance of a credential.
+
+use crate::SupportedCredentialType;
+use std::fmt::{Display, Formatter};
+use vc_util::issuer_api::{
+    Icrc21ConsentInfo, Icrc21ConsentMessageResponse, Icrc21ConsentPreferences,
+};
+use SupportedCredentialType::{UniversityDegreeCredential, VerifiedEmployee};
+use SupportedLanguage::{English, German};
+
+const EMPLOYMENT_VC_DESCRIPTION_EN: &str = r###"# DFINITY Foundation Employment Credential
+
+Credential that states that the holder is employed by the DFINITY Foundation at the time of issuance."###;
+const DEGREE_VC_DESCRIPTION_EN: &str = r###"# Bachelor of Engineering, DFINITY College of Engineering
```

Comment:
Yes, however for that it would make sense to use areal templating engine. `format!` unfortunately _requires_ a string literal.

Should I pull in a templating engine?

Reply:
Sure, I don't mind adding a (not-too-complex) dependency if that enables the templating.

Reply:
maybe https://crates.io/crates/tinytemplate would be suitable?

Reply:
I opted for https://crates.io/crates/strfmt because it seems even tinier.